po/it.po

```diff
@@ -514,20 +505,20 @@ msgstr "Bordo intorno all'immagine in modalità Camera Oscura"
 #: ../src/lua/preferences.c:731
 #, c-format
 msgid "double click to reset to `%d'"
-msgstr "Doppio click per ripristinare a '%d'"
+msgstr "Doppio clic per ripristinare a '%d'"
 
 #: ../build/bin/preferences_gen.h:4639
 msgid ""
-"process the image in darkroom mode with a small border. set to 0 if you "
-"don't want any border."
+"process the image in darkroom mode with a small border. set to 0 if you don't "
+"want any border."
 msgstr ""
-"Elabora l'immagine in modalità Camera Oscura con un bordo piccolo. Imposta a "
-"0 se non vuoi alcun bordo."
+"Elabora l'immagine in modalità 'Camera oscura' con un bordo piccolo. \n"
```

Comment:
Per convenzione abbiamo concordato di usare Camera Oscura (senza virgolette) e Tavolo Luminoso con la maiuscola per identificare i rispettivi moduli. L'apostrofo non mi pare corretto.

Reply:
Una regola base della grammatica italiana impone che le lettere maiuscole si usano solo per la prima lettera di una frase ad eccezione dei nomi propri.
Sono in inglese che usano le lettere maiuscole per ogni parola. Ma non vale per l'italiano.
Quindi Camera Oscura non va bene in italiano ma dovrebebe essere Camera oscura.
Idem per Tavolo luminoso.
L'apostrofo prima e dopo indica nelle descrizoni il riferimento ad una specifica funzione/opzione.